head/firmware/CMakeLists.txt

```diff
@@ -144,9 +144,9 @@ add_custom_target(head-debug
         )
 
 
-# Runs openocd to flash the board (without using a debugger)
+# A target to flash bootloader and firmware
 add_custom_target(head-flash
-        COMMAND "${OpenOCD_EXECUTABLE}" "-f" "${COMMON_EXECUTABLE_DIR}/STM32G491RETx/stm32g4discovery.cfg" "-c" "program $<TARGET_FILE:head>;reset;exit"
+        COMMAND "${OpenOCD_EXECUTABLE}" "-f" "${COMMON_EXECUTABLE_DIR}/STM32G491RETx/stm32g4discovery.cfg" "-c" "init;reset init;halt;stm32g4x mass_erase 0;flash write_image $<TARGET_FILE_DIR:bootloader-head>/bootloader-head.hex;flash write_image head.hex;reset;exit"
```

Comment:
I think it might be nice/more readable to make the stuff past the `-c` a template file and configure the file in `cmakelists.txt` (similar to the gdbinit file).

Reply:
Good idea. I'll check it out.